inbm/cloudadapter-agent/tests/unit/cloud/adapters/test_inbs_adapter.py

```diff
@@ -0,0 +1,27 @@
+"""
+Unit tests for the InbsAdapter class
+
+
+"""
+
+import unittest
+
+from time import time
+
+from cloudadapter.cloud.adapters.inbs_adapter import InbsAdapter
+
+
+class TestInbsAdapter(unittest.TestCase):
+
+    def setUp(self) -> None:
+        self.CONFIG: dict[str, str] = {
+            "hostname": "localhost",
+            "port": "50051",
+            "node-id": "node_id",
+            "token": "token",
+        }
+
+        self.inbs_adapter = InbsAdapter(self.CONFIG)
+
+    def test_configure_succeeds(self) -> None:
+        self.inbs_adapter.configure(self.CONFIG)
```

Comment:
**<h2>$\color{#15F4EE}Miscellaneous$</h2>**
**Suggestion:** Add assertions to verify the behavior of configure method in InbsAdapter.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
    def test_configure_succeeds(self) -> None:
        self.inbs_adapter.configure(self.CONFIG)
        # Verify that the adapter is correctly configured
        self.assertEqual(self.inbs_adapter.hostname, self.CONFIG["hostname"])
        self.assertEqual(self.inbs_adapter.port, self.CONFIG["port"])
        self.assertEqual(self.inbs_adapter.node_id, self.CONFIG["node-id"])
        self.assertEqual(self.inbs_adapter.token, self.CONFIG["token"])
```